doc/classes/HBoxContainer.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="HBoxContainer" inherits="BoxContainer" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Horizontal box container.
+		A container that arranges its child controls horizontally.
 	</brief_description>
 	<description>
-		Horizontal box container. See [BoxContainer].
+		A container that arranges its child controls horizontally, rearranging them automatically when their minimum size changes.
```

Comment:
It could also say that its a variation of BoxContainer that works only horizontally.

Reply:
I'm not sure about this. The brief description should be explanatory and not link to some other class, expecting you to read the explanation there. Adding this would make the brief one too long, especially in harder-to-explain classes like FlowContainer. If you insist, I think the full description is a better place for it.

Reply:
Well my comment is under description...
```suggestion
		[HBoxContainer] is a variation of [BoxContainer] that works only horizontally. It automatically rearranges its [Control] children when their minimum size changes.
```
Something like that maybe.